### ISSUE TITLE
Learning rate schedule

### ISSUE THREAD

**Issue by antoniomastro1996** [google-research/text-to-text-transfer-transformer#218]
Hi all,

I have the following doubt: as far as I understood from the paper, during fine-tuning you have used a constant learning rate of 0.001. However, from the code you shared on colab, I can see that

model = t5.models.MtfModel(
    model_dir=MODEL_DIR,
    tpu=TPU_ADDRESS,
    tpu_topology=TPU_TOPOLOGY,
    model_parallelism=model_parallelism,
    batch_size=train_batch_size,
    sequence_length={"inputs": 128, "targets": 32},
    <b>learning_rate_schedule=0.003,</b>
    save_checkpoints_steps=5000,
    keep_checkpoint_max=keep_checkpoint_max if ON_CLOUD else None,
    iterations_per_loop=100,
)

Is this learning_rate_schedule refers to the same thing I refer to or is something different?
In other work If I want to reproduce exactly your configuration is sufficient to set a learning_rate_schedule = 0.001 instead of 0.003?

**Comment by adarob** [comment]
Yes, I'm not sure why I used 0.003 in the colab, but 0.001 is what we used in the paper.
